v2alpha1/api/swagger/components/parameters/Next.yaml

```diff
@@ -0,0 +1,7 @@
+name: next
+in: query
+description: Specifies the pagination cursor for the next page of resources
+required: false
+schema:
```

Comment:
if next is specified, then API must return the first page. Document this?

Reply:
It doesn't actually returns the first page, but the page "pointed" by next

Reply:
sorry. I wanted to say 'if next **is not** specified'.